### PR TITLE
Added list of RP2040-based boards.

### PR DESCRIPTION
**Pull request by aallan**
There is a problem loading the images into the HTML. See the associated issue #2057. 

**NOTE:** Do not merge until #2057 is resolved.

### CODE REVIEW

File: documentation/asciidoc/microcontrollers/rp2040/rp2040_based_boards.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+=== Boards from Arduino
+
+image::images/arduino_nano.png[]
+
+Nano RP2040 Connect:: Meet the only connected RP2040 board, leveraging essential key features of the other Arduino Nanos — like the versatile u-blox NINA WiFi and Bluetooth module. The new board comes packed with some high-quality MEMS sensors from STM, an efficient power section, and a bunch of other innovations.
+
+=== Boards from Pimoroni
+
+image::images/pga2040.png[]
```
Comment (lurch) [inline documentation/asciidoc/microcontrollers/rp2040/rp2040_based_boards.adoc:27]: This needs to be `images/pga2040.jpg`

Comment (aallan) [reply documentation/asciidoc/microcontrollers/rp2040/rp2040_based_boards.adoc]: Fixed in 5781295. But they're not **all** wrong, and none of the images are copied to the HTML directory?

Comment (aallan) [reply documentation/asciidoc/microcontrollers/rp2040/rp2040_based_boards.adoc]: Changing `.png` to `.jpg` didn't help, which it certainly should have done! Also, thinking about it. Shouldn't that have thrown an error when it *tried* to copy the file to the HTML directory and couldn't find it? There wasn't an error generated.

**Comment by aallan** [comment]
This should be fine to merge now #2070 fixes #2057.

**Comment by aallan** [comment]
...and can confirm #2070 fixes the problem! 😄

File: documentation/asciidoc/microcontrollers/rp2040/rp2040_based_boards.adoc (hunk 1 above)
Comment (lurch) [reply documentation/asciidoc/microcontrollers/rp2040/rp2040_based_boards.adoc]: > Shouldn't that have thrown an error when it tried to copy the file to the HTML directory and couldn't find it?

It **did** throw an error about the missing file when I was fixing #2070 and that's how I spotted that this filename was wrong :wink:

**Pull request merged**